blocks/library/quote/index.js

```diff
@@ -26,36 +26,38 @@ const fromEditableValue = value => value.map( ( subValue ) => ( {
 	children: subValue,
 } ) );
 
+const blockAttributes = {
+	value: {
+		type: 'array',
+		source: 'query',
+		selector: 'blockquote > p',
+		query: {
+			children: {
+				source: 'node',
+			},
+		},
+		default: [],
+	},
+	citation: {
+		type: 'array',
+		source: 'children',
+		selector: 'cite',
+	},
+	align: {
+		type: 'string',
+	},
+	style: {
+		type: 'number',
+		default: 1,
+	},
+};
+
 registerBlockType( 'core/quote', {
 	title: __( 'Quote' ),
 	icon: 'format-quote',
 	category: 'common',
 
-	attributes: {
```

Comment:
Why move this away?

Reply:
Just to reuse it for the deprecated version since most attributes are shared.

Reply:
I have concerns with block looking different at first sight when you open one to learn from, that i think we should minimize.